mongodbatlas/resource_mongodbatlas_project_api_key.go

```diff
@@ -191,17 +191,6 @@ func resourceMongoDBAtlasProjectAPIKeyUpdate(ctx context.Context, d *schema.Reso
 	projectID := ids["project_id"]
 	apiKeyID := ids["api_key_id"]
 
-	updateRequest := new(matlas.AssignAPIKey)
-	if d.HasChange("role_names") {
```

Comment:
root role_names was removed in [INTMDB-1015](https://jira.mongodb.org/browse/INTMDB-1015)